.github/workflows/ci-test.yml

```diff
@@ -1,17 +1,33 @@
-name: CI
+name: Automated tests
 
 on:
   push:
     branches: [ main ]
+    paths:
+        - 'package.json'
+        - 'src/**/*.html'
+        - 'src/**/*.js'
+        - 'src/**/*.json'
+        - 'tests/**/*.html'
+        - 'tests/**/*.js'
+        - 'tests/**/*.json'
   pull_request:
     branches: [ main ]
+    paths:
+        - 'package.json'
+        - 'src/**/*.html'
+        - 'src/**/*.js'
+        - 'src/**/*.json'
+        - 'tests/**/*.html'
+        - 'tests/**/*.js'
+        - 'tests/**/*.json'
```

Comment:
I think we should also add the Gruntfile.js to the list of tracked files. Possibly also the .nycrc file as that configures the code coverage reporting.